components/entitlement/org.wso2.carbon.identity.api.server.entitlement/pom.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license header.

Reply:
```suggestion
  ~ Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com).
```

Reply:
I don't think we will be able to do this as this is a moving of lot of files.

Reply:
Done